### PR TITLE
Extend columns on application table

### PR DESCRIPTION
**Pull request by colinbruce**
[Trello ticket](https://trello.com/c/PpcKSV4z/6-update-list-of-process-and-deleted-applications-to-include-marked-for-evidence-check-and-part-payment)
[Original pivotal ticket](https://www.pivotaltracker.com/story/show/113885717)

Add extra columns to the processed and deleted application tables

## To do
- [x] Code solution
- [x] Investigate styling

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8f877e9914b3193edd9f84ae71c35eb74ecd6437
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/ministryofjustice/fr-staffapp

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: add16e81811ffa016146db291edbc825d9d32dc2
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/ministryofjustice/fr-staffapp

---
This message was auto-generated by https://gitcop.com

**Comment by axemonkey** [comment]
I wasn't really able to add much, other than stopping the reference numbers and dates from wrapping, and adding align-top to the rows. Oh and decreasing the right padding. I don't know if this will make much difference, but then you do have a lot of columns now. It may - as suspected - be necessary to reduce font size to get everything in.

File: config/locales/en.yml [hunk 1]
```diff
@@ -25,7 +28,9 @@ en-GB:
       processed_on: Date processed
       form_name: Form
       fee: Fee
+      part_payment: Part- payment
```
Comment (doctorpod) [inline config/locales/en.yml]: Did you mean to put a space after the dash here?

Comment (colinbruce) [inline config/locales/en.yml]: Good catch... originally yes, to force a wrap of the text, before Clive worked his magic...

**Pull request merged**